kotsadm/pkg/apiserver/server.go

```diff
@@ -102,6 +101,11 @@ func Start() {
 	* Session auth routes
 	**********************************************************************/
 
+	sessionAuthQuietRouter := r.PathPrefix("").Subrouter()
+	sessionAuthQuietRouter.Use(handlers.RequireValidSessionQuietMiddleware)
+
+	sessionAuthQuietRouter.Path("/api/v1/ping").Methods("GET").HandlerFunc(handlers.Ping)
```

Comment:
can you remove this comment? https://github.com/replicatedhq/kots/blob/master/kotsadm/pkg/handlers/ping.go#L18